client/coral-embed/index.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-var */
+/**
+ * @file Exposes the main 'window.Coral.Talk' API that developers can use to
+ * render Talk streams in their webpages
+ * @todo Currently implemented to be included directly via <script> tag
+ * (including copypasta dependencies like pym.js), but later there will be a
+ * build step and this code may use import statements 
+ */
+// using umd.js (https://github.com/umdjs/umd/blob/master/templates/returnExports.js)
+(function (root, factory) {
+  /* eslint-disable */
+  if (typeof define === 'function' && define.amd) {
+
+    // AMD. Register as an anonymous module.
+    define([], factory);
+  } else if (typeof module === 'object' && module.exports) {
+
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory();
+  } else {
+
+    // Browser globals (root is window)
+    root.Coral = factory();
+  }
+  /* eslint-enable */
+}(this, function () {
+
+  // This function should return value of window.Coral
+  let pym = requirePym();
+  let Coral = {};
+  let Talk = Coral.Talk = {};
+
+  /**
+   * Render a Talk stream
+   * @param {HTMLElement} el - Element to render the stream in
+   * @param {Object} opts - Configuration options for talk
+   * @param {String} opts.talk - Talk base URL
```

Comment:
It may be desirable to not require this here, but it's needed to build the iframe URL. It could be added to this script at build time (or render of the .js time), but right now this is just a static script so it needs to be provided by user at time of use.